lib/decanter/parser/string_parser.rb

```diff
@@ -1,10 +1,10 @@
 module Decanter
   module Parser
     class StringParser < ValueParser
-
-      allow String
-
       parser do |val, options|
+        raise Decanter::ParseError if val.is_a? Array
+        next if (val.nil? || val === '')
```

Comment:
I don't love this behavior but it's necessary for now to avoid a major breaking change (passing through empty string values).